utils/bazel/.bazelrc

```diff
@@ -2,10 +2,10 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-build --action_env=CC=clang
-build --action_env=CXX=clang++
-build --cxxopt=-std=c++17 
-build --host_cxxopt=-std=c++17 
+build --action_env=CC=clang-10
+build --action_env=CXX=clang-cpp-10
```

Comment:
@asaadaldien PTAL. In the past we'd decided to not send this upstream because clang on GHA VMs was already new, but with Ubuntu 18 container the default clang is pretty old and fails with the AVX errors.